### PR TITLE
refactor(@angular/cli) : update dependancies version

### LINKED ISSUE #92: Update dependencies and clean package.json
Follow update.angular.io in order to update to following dependencies to their latest versions:
- @angular/cli
- @angular/core
- @angular/cdk
- @angular/material
- @ngrx/store
- rxjs

Verify potential conflicts with step branches, put all the required dependencies in master, and then clean unused dependencies.

### PR DESCRIPTION
**Pull request by geromegrignon**
add angular.json
delete angular-cli.json
update dependancies
update project name to angular-200 into files

closes #92

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -0,0 +1,128 @@
+{
+  "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
+  "version": 1,
+  "newProjectRoot": "projects",
+  "projects": {
+    "angular-200": {
```
[the remark this answers is not in the document]
Comment (geromegrignon) [reply angular.json]: already working on project architecture but wanted to focus strictly on the #92 issue for this pull request

[the remark this answers is not in the document]
Comment (noelmace) [reply angular.json]: Could you add a comment on #93 in order to not forget that then ? Thanks

[the remark this answers is not in the document]
Comment (noelmace) [reply angular.json]: Could you add a comment on #93 in order to not forget that then ? Thanks

File: tsconfig.json [hunk 2]
```diff
@@ -15,6 +15,7 @@
     "lib": [
```
[the remark this answers is not in the document]
Comment (geromegrignon) [reply tsconfig.json]: I used update.angular.io rules with ng update. I tried once again (and with another project too).
ng update@angular/core only updates package.json.
I changed tsconfig manually to es2018

**Comment by noelmace** [comment]
FYI, it seems that some of your commits where created with an author adresse you forgot to add to your github account.

File: package.json [hunk 3]
```diff
@@ -19,48 +19,50 @@
...
-    "@angular/router": "5.0.1",
-    "@ngrx/router-store": "5.0.1",
-    "@ngrx/store": "5.0.0",
-    "body-parser": "1.15.2",
-    "core-js": "^2.4.1",
-    "cors": "2.7.1",
-    "es6-shim": "0.35.1",
-    "express": "4.14.0",
+    "@angular/animations": "~7.0.0",
+    "@angular/cdk": "^7.0.1",
```
[the remark this answers is not in the document]
Comment (noelmace) [reply package.json]: not resolved

File: angular.json [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+      "root": "",
```
[the remark this answers is not in the document]
Comment (noelmace) [reply angular.json]: Not src/project, project.

[the remark this answers is not in the document]
Comment (geromegrignon) [reply angular.json]: You're right! I had little  knowledge about project architecture few days ago.
it'll be like that : 

```
"step1": {
      "root": "projects/step1/",
      "sourceRoot": "projects/step1/src",
...
```

**Pull request merged**